### DISCUSSION TITLE
请问下SDK支持导出PNG，或者讲PAG格式转为webp或视频吗？

### DISCUSSION THREAD

**Discussion by Alexhu-xp** [Tencent/libpag#2546] [Q&A] [answered]
请问下SDK支持导出PNG，或者讲PAG格式转为webp或视频吗？

**Comment by kevingpqi123** [comment, marked as answer]
看下这个 demo： https://github.com/Tencent/libpag/blob/main/linux/src/main.cpp
pag 支持获取每一帧渲染数据，如 RGBA 格式，然后自己编码成 PNG、webp或视频
